wikimedia/wikipedia-ios
# Image recommendations - Improve Dynamic Type support

**Phabricator:** https://phabricator.wikimedia.org/T358930 & https://phabricator.wikimedia.org/T363929

### Notes
This PR improves dynamic type support in the image recommendations flow. Updates are:

1. Fixed bottom sheet title view clipping, added scroll view to scroll to see all content.
2. Added scrolling within tooltips.
3. Allowed header and footer views to scale in `InsertMediaSettingsViewController`.
4. Improve wrapping on the toggle views in `EditSaveViewController`. 

### Test Steps
Tap through Image Recommendations flow on the largest dynamic type size. Ensure content is not clipped.
